mythril/analysis/report.py

```diff
@@ -64,6 +63,24 @@ def __init__(
         self.source_mapping = None
         self.discovery_time = time() - StartTime().global_start_time
         self.bytecode_hash = get_code_hash(bytecode)
+        if transaction_sequence is None:
+            self.tx_sequence_users = None
+            self.transaction_sequence_jsonv2 = None
+        else:
+            self.tx_sequence_users = json.dumps(transaction_sequence, indent=4)
+            self.transaction_sequence_jsonv2 = self.add_block_data(transaction_sequence)
+
+    @staticmethod
+    def add_block_data(transaction_sequence: Dict):
```

Comment:
Shouldn't this be done by the logic that creates the concrete transaction sequence?

Reply:
But we need to use this from symbolic tx sequence stuff right?

Reply:
What do you mean?

Reply:
We don't have the concrete data of these gasLimit, blockNumber etc right?

Reply:
The blocknumber could be constrained right?

Reply:
Same for some of the other variables, they are part of the execution environment and can be queried through some of the opcodes

Reply:
yes, but I wanted to do that after this gets merged, there are other things too which I skipped at the moment, I wanted to create a new issue for them to be tackled after the merge. (Like if an issue pops up  in creation tx etc)

Reply:
Sounds reasonable :+1: